### DISCUSSION TITLE
Uncaught TypeError: $.map is not a function

### DISCUSSION THREAD

**Discussion by pk-pankaj** [breezefront/community#203] [3. Integrations]
Hi @vovayatsyuk 

We are integrating a third party module webkul giftcard and there is an error

giftcardview.js:278 Uncaught TypeError: $.map is not a function

`timeZone.html($.map(self.options.timeZones, function(elem){
                return $('<option>', {text: elem.label, value: elem.value});
            }));`

the module is using like above, so is there any alternate of this in Breeze or how we can resolve this ?

**Comment by vovayatsyuk** [comment]
It depends on what type is `self.options.timeZones`.

If it's an array, you can use the [native array.map function](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map).

If it's an object, you can replace it [with an underscore's map function](https://underscorejs.org/#map).
